samila/functions.py

```diff
@@ -27,17 +27,22 @@ def random_equation_gen():
 
     :return: equation as str
     """
-    num_elements = random.randint(2, len(ELEMENTS_LIST) + 3)
```

Comment:
Why did we do that earlier? :thinking:

Reply:
In the early versions of the `random_equation_gen` function, the length of `ELEMENTS_LIST` was about 2-3, with the addition of more options to this list we need to reduce the maximum number of elements (to balance complexity and beauty)